projects/storefrontlib/src/layout/main/main.module.ts

```diff
@@ -24,6 +25,7 @@ import { StorefrontComponent } from './storefront.component';
     PageSlotModule,
     AnonymousConsentsModule,
     FeaturesConfigModule,
+    SkipModule,
```

Comment:
Keep in mind that going forward we probably want to import a single `AccessibilityModule` (similar to importing the `SeoModule`). NP for now.